src/engine/specification.md

```diff
@@ -470,15 +470,24 @@ This method follows the same specification as [`engine_forkchoiceUpdatedV1`](#en
 * method: `engine_getPayloadV2`
 * params:
   1. `payloadId`: `DATA`, 8 Bytes - Identifier of the payload build process
+* timeout: 1s
 
 #### Response
 
-* result: [`ExecutionPayloadV2`](#ExecutionPayloadV2)
+* result: `object`
+  - `executionPayloadV2`: [`ExecutionPayloadV2`](#ExecutionPayloadV2)
+  - `blockValue` : `QUANTITY`, 256 Bits - The expected value to be recieved by the `feeRecipient` in wei
 * error: code and message set in case an exception happens while getting the payload.
 
 #### Specification
 
-Refer to the specification for [`engine_getPayloadV1`](#engine_getpayloadv1).
+1. Given the `payloadId` client software **MUST** return the most recent version of the payload that is available in the corresponding build process at the time of receiving the call.
```

Comment:
Instead of copying full specification you may put it in the following way:
#
This method follows the same specification as [`engine_getPayloadV1`](#engine_getpayloadv1) with the addition of the following:

1. Client software **MAY** use the sum of the block's priority fees to determine `blockValue`.